---
title: "ATACgraph"
url: "https://github.com/RitataLU/ATACgraph"
scope: "domain"
platforms:
  - platform_group: "container"
    platform_url: "https://hub.docker.com/r/lsbnb/galaxy_atacgraph"
    platform_text: "ATACgraph Docker"
summary: "ATAC-seq for biologists with minimal bioinformatics knowledge to analyze chromatin accessibility"
image: "atacgraph-flow.png"
comments:
  - "ATACgraph profiles accessible chromatin regions and provides ATAC-seq-specific information including definitions of nucleosome-free regions (NFRs) and nucleosome-occupied regions. ATACgraph also allows identification of differentially accessible regions between two ATAC-seq datasets."
user_support:
  - "[Installation instructions](https://hub.docker.com/r/lsbnb/galaxy_atacgraph)"
  - "Predefined workflows"
  - "[Issue reporting](https://github.com/RitataLU/ATACgraph/issues)"
quotas:
  - "None"
citations:
  - "Lu, R. J.-H., Liu, Y.-T., Huang, C. W., Yen, M.-R., Lin, C.-Y., & Chen, P.-Y. (2021). [ATACgraph: Profiling Genome-Wide Chromatin Accessibility From ATAC-seq](https://doi.org/10.3389/fgene.2020.618478). *Frontiers in Genetics*, 11. doi: 10.3389/fgene.2020.618478"
pub_libraries:
  - "ATACgraph"
sponsors:
  - "[Institute of Plant and Microbial Biology](https://ipmb.sinica.edu.tw/) and [Institute of Information Science](https://www.iis.sinica.edu.tw/), [Academia Sinica](https://www.sinica.edu.tw/), Taipei, Taiwan"
  - "[Department of Medicine](https://internalmedicine.wustl.edu/), [Washington University in St. Louis](https://wustl.edu/), St. Louis, MO, United States"
---
